process/living-cr-rec.html

```diff
@@ -0,0 +1,145 @@
+            alt="W3C" height="48" width="72"/></a></span>
+      <div class="breadcrumb"> <a href="https://www.w3.org/Guide/">The Art of
+          Consensus</a> →
+        <h1>Candidate Recommendation Snapshots and Recommendations</h1>
+      </div>
+      <p class="baseline">This <strong>Guidebook</strong> is the collected
+        wisdom of the W3C Group Chairs and other collaborators.</p>
+    </div>
+    <!-- <div class="toc">
+      <h4>Also On This Page → </h4>
+      <ul>
+        <li><a href="#prep">Preparatory Work</a> •</li>
+        <li><a href="#wg-startup">Starting the Group</a> •</li>
+        <li><a href="#fpwd">First Public Working Draft</a> •</li>
+        <li><a href="#test">Test suites</a> •</li>
+        <li><a href="#getting-to-lc">Getting to Candidate Recommendation</a> •</li>
+        <li><a href="#processing-lc">Processing Candidate Recommendation</a> •</li>
+        <li><a href="#transition-cr">Transition to CR</a> •</li>
+        <li><a href="#leaving-cr">Leaving CR</a> •</li>
+        <li><a href="#pr">Proposed Recommendation</a> •</li>
+        <li><a href="#rec">Recommendation</a> •</li>
+      </ul>
+    </div> -->
+    <div class="main">
+      <h2 id="intro">Introduction</h2>
+
+      <p>
+        This document is intended to help Working Groups in deciding whether to
+        maintain a document using revised Candidate Recommendations or revised Recommendations.
+      </p>
+      <h2 id="process">W3C Process</h3>
+      <p>The W3C Process offers two ways to maintain a document on the Recommendation track:</p>
+      <dl>
+        <dt><a href="https://www.w3.org/Consortium/Process/#revising-cr">Revising a Candidate Recommendation</a></dt>
+        <dd>
+          Candidate Recommendation Snapshots get similar patent commitments to W3C Recommendations but do not get
+          endorsed by W3C as standards for the Web. The document can be revised rapidly by publishing Candidate
+          Recommendation Drafts on a daily basis.
+        </dd>
+        <dt><a href="https://www.w3.org/Consortium/Process/#revising-rec">Recommendations</a></dt>
+        <dd>
+          Recommendations are endorsed by W3C as standards for the Web. The document can be revised rapidly by publishing candidate
+          corrections/additions on a daily basis.
+        </dd>
+      </dl>
+
+      <h2 id="cr">When should a Group keep a document as a Candidate Recommendation?</h2>
+
+        <p>
+          A document remains in Candidate Recommendation if one or more of the conditions apply:
+        </p>
+        <ol>
+          <li>Unresolved issues that prevent any meaningful/core subset of the document to move forward</li>
+          <li>Insufficient test coverage to demonstrate adequate implementation experience</li>
+          <li>Insufficient adequate implementation experience</li>
+        </ol>
+
+        <p>If unresolved technical issues remain for long period of time due to the impossibility of finding technical
```

Comment:
```suggestion
        <p>If unresolved technical issues remain for long period of time due to the impossibility of finding technical
        <p>If technical issues remain unresolved for a long time,
```
(see next suggestion because I haven't figured out how to make them cover more than one line yet...

Also, there should be a distinction between stopping work becausee you can't solve any problems, and shipping something that solves important problems even if there are others outstanding - unless they are all so tangled that you can't be good without being perfect.